guardian/prosemirror-typerighter
# Add match context to model

## What does this change?
Adds a  'matchContext' field to the IMatch type so that we can pass it back out to the feedback form.

## How to test

Install on local compower and run local typerighter. This has been done successfully.

See also: https://github.com/guardian/typerighter/pull/59

## How can we measure success?
Feedback form has matchContext in the technical data section.

## Have we considered potential risks?
No

## Images
N/A
